icepyx/core/query.py

```diff
@@ -449,10 +60,6 @@ def __init__(
                 self._prod, cycles=self.cycles, tracks=self.tracks
             )
 
-        # initialize authentication properties
-        EarthdataAuthMixin.__init__(self)
-
-    # ----------------------------------------------------------------------
```

Comment:
I can't tag the lines specifically, but why do we need these properties (`product`, `product_version`, `__str__`) in both Query and BaseQuery? Could we have them in just one or the other location?

Reply:
I de-duplicated these methods, no need to re-implement them if we are not doing anything different.